packages/manager/modules/web-paas/src/details/service/change-offer/change-offer.controller.js

```diff
@@ -59,10 +60,7 @@ export default class {
 
   scrollToTop() {
     this.$timeout(() => {
-      document.getElementById('web-pass-add-header').scrollIntoView({
-        behavior: 'smooth',
-      });
-      document.getElementById('web-pass-add-alert').focus();
+      document.getElementById('web-pass-add-header').scrollIntoView(false);
```

Comment:
scrollToTop() {
this.$location.hash(this.PAGE_SECTIONS.BACK_BUTTON);
this.$anchorScroll();
}